src/main/resources/dashboard/repo.hbs

```diff
@@ -71,6 +69,55 @@ only be able to download them. Nobody else will have any access to the repo.</p>
 <a href="https://maven.apache.org/plugins/maven-deploy-plugin/usage.html"><code>mvn deploy</code></a>
 and you download with
 <a href="https://maven.apache.org/plugins/maven-compiler-plugin/index.html"><code>mvn compile</code></a>.
+{{/eq}}{{#eq type "maven-proxy"}}
+<p>It's maven proxy repository, it proxies all request to remote Maven repository apply authentication
+and save artifacts in Artipie storage cache.</p>
+
+<p>To use it as a mirror for Apache central, add this configuration to your
+<code>~/.m2/settings.xml</code>:
+<pre>
+&lt;settings&gt;
+  [...]
+  &lt;mirrors&gt;
+    &lt;mirror&gt;
+      &lt;id&gt;artipie-mirror&lt;/id&gt;
+      &lt;name&gt;Artipie Mirror Repository&lt;/name&gt;
+      &lt;url&gt;https://central.artipie.com/{{user}}/{{repo}}&lt;/url&gt;
+      &lt;mirrorOf&gt;central&lt;/mirrorOf&gt;
+    &lt;/mirror&gt;
+  &lt;/mirrors&gt;
+&lt;/settings&gt;
+</pre>
+</p>
+{{/eq}}{{#eq type "docker"}}
+<p>Tag your image with <code>central.artipie.com/{{user}}/{{repo}}</code> image prefix,
+and push it to central.artipie.com then. E.g.
+for <code>alpine:3.11</code> use:
+<pre>
+docker tag alpine:3.11 central.artipie.com/{{user}}/{{repo}}alpine:3.11
+docker push central.artipie.com/{{user}}/{{repo}}alpine:3.11
+</pre>
+</p>
+{{/eq}}
+{{#eq type "maven-group"}}
+To use Maven groupped (virtual) repository, you have to create at least two Maven repositories before,
+it can be either just a Maven repository or Maven proxy (mirror) repository. Then specify
```

Comment:
@g4s8 I'd suggest to rephrase:
To use Maven grouped (virtual) repository first create at least two Maven or Maven proxy repositories.